web/src/typeahead_helper.ts

```diff
@@ -104,6 +104,7 @@ export let render_typeahead_item = (args: {
     is_user_group?: boolean;
     stream?: StreamData;
     emoji_code?: string | undefined;
+    render_along_with_topic?: boolean;
```

Comment:
Instead of this, try passing the whole topic object in and adding the topic html in that handlebars codeblock, instead of having to call two different render functions above and concatenate them together.